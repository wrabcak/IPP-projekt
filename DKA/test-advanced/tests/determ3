(
{S,fiNaL,Q1,q2},
{'A', 'b', 'c'},
{
s'a' -> s,
S'b' -> q1, 
s'b' -> FinAl, 
S'C' -> q2, 
s'c' -> FinaL,
Q1'b' -> q1, 
q1'B' -> finAL, 
q2'C' -> Q2, 
Q2'c' -> final , 
finAL'A' -> FINAL
},
s,
{FiNAl}
)
