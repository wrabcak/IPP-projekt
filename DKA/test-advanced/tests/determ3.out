(
{final, final_q1, final_q2, s},
{'a', 'b', 'c'},
{
final 'a' -> final,
final_q1 'a' -> final,
final_q1 'b' -> final_q1,
final_q2 'a' -> final,
final_q2 'c' -> final_q2,
s 'a' -> s,
s 'b' -> final_q1,
s 'c' -> final_q2
},
s,
{final, final_q1, final_q2}
)